jest.config.js

```diff
@@ -3,7 +3,12 @@ const path = require('path')
 module.exports = {
   verbose: true,
   collectCoverageFrom: [
-    '**/src/**/**/*.js'
+    '**/src/**/**/*.js',
+    '!src/server.js',
+    '!src/handlers/index.js',
+    '!src/handlers/init.js',
+    '!src/lib/logger/*.js',
+    '!src/lib/startingProcess.js'
```

Comment:
I focused on adding unit-tests, first of all, for business logic. So that I could start testing my changes with real Kafka and DB earlier (to be sure that I didn't introduce any regression). But if we have enough time to cover all these "helper" functionalities, let's do it, I don't mind.

Reply:
Yes, we need to add test coverage for all the code we added and we shouldn't exclude any file when we merge to master.

Reply:
Ok, clear. Added a new task:
![Uploading image.png…]()

Reply:
Sure I am ok with the new task. You can add the coverage to the same PR.